packages/babel/src/preflightCheck.js

```diff
@@ -34,7 +34,7 @@ function helpersTestTransform() {
 const mismatchError = (actual, expected, filename) =>
   `You have declared using "${expected}" babelHelpers, but transforming ${filename} resulted in "${actual}". Please check your configuration.`;
 
-const inheritsHelperRe = /\/helpers\/(esm\/)?inherits/;
+const inheritsHelperRe = /[\\/]+helpers[\\/]+(esm[\\/]+)?inherits/;
```

Comment:
Hm, I'm rather surprised that this would be needed. I have never seen a windows path delimiter in the import specifier. Isn't this normalized by Babel and all the tools to use POSIX delimiter? 

Do you have a repro case of the problem? I would love to take a look at that.

Reply:
Ok, I've found out the repro case in the linked ticket - thanks for that and for the detailed description of the problem there.

I think that the best thing to do would be to first try using [`normalize-path`](https://github.com/jonschlinkert/normalize-path) on this computed path:
https://github.com/facebook/create-react-app/blob/a422bf227cf5294a34d68696664e9568a152fd8f/packages/babel-preset-react-app/create.js#L51-L55

If a PR implementing such a change won't land in CRA cause their maintainers won't have time/bandwidth to handle this then we could consider proceeding with this PR here.

You can also try disabling the `absoluteRuntime` option in your `babel-preset-react-app`'s options. I didn't have time to dig into why this is even the default etc but I think that for the vast majority of use cases this actually shouldn't be used.

Reply:
Ok - I'll create a PR for them and reference this issue, and we'll see where it takes us. I the meantime, I'll just skip the preflight checks.

Reply:
I dug into this a little bit more... seems like the windows-style import comes from https://github.com/babel/babel/blob/6ec66d8d2249e345a587d8d47722abd779be6416/packages/babel-plugin-transform-runtime/src/index.ts#L241. The absoluteRuntime config _value_ isn't used in this computation, so changes in CRA has no effect.

So the question is then - should we try to land a change in babel-plugin-transform-runtime, or stick with the regex fix in this PR?

Reply:
> So the question is then - should we try to land a change in babel-plugin-transform-runtime, or stick with the regex fix in this PR?

I'm leaning towards... both 😅 I would appreciate it if you could:
- prepare a PR to Babel
- add a comment in the code about this issue and link to the created PR